iothub/service/src/DigitalTwin/Models/UpdateDigitalTwinResponse.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// The service response to a digital twin update operation.
+    /// </summary>
+    public class UpdateDigitalTwinResponse
+    {
+        /// <summary>
+        /// Initializes a new instance of the
+        /// DigitalTwinUpdateHeaders class.
```

Comment:
Overly aggressive line length and copy/pasta error (?) of class name.

```suggestion
        /// Initializes a new instance of this class.
```